pkg/util/utils.go

```diff
@@ -682,18 +682,21 @@ func DefaultContainerConfig() *config.Config {
 	return containerConfig
 }
 
-func CreateCidFile(cidfile string, id string) error {
-	cidFile, err := OpenExclusiveFile(cidfile)
-	if err != nil {
-		if os.IsExist(err) {
-			return fmt.Errorf("container id file exists. Ensure another container is not using it or delete %s", cidfile)
+func CreateIDFile(path string, id string) error {
+	baseDir := filepath.Dir(path)
+	if baseDir != "" {
+		if _, err := os.Stat(baseDir); err != nil {
+			return err
```

Comment:
Why stat the dir before? Either open will work or not I do not see what value this call is adding?

Reply:
No idea.  I preserved the previous logic but feel more comfortable now to just change that.